test/wg-easy/1.0.0/templates/delete-route-post-delete-hook.yaml

```diff
@@ -0,0 +1,33 @@
+{{ $values := (. | mustDeepCopy) }}
+{{ $_ := set $values "common" (dict "nameSuffix" "wg-easy") }}
+{{ $ip := .Values.wgeasy.client_address_range | replace "x" "0" }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: post-delete-routes-wg-easy
+  annotations:
+    "helm.sh/hook": post-delete
```

Comment:
This would only run when the app is deleted, if the app is scaled down we don't want leftover routes by wireguard lying around.

Reply:
I've left the post-delete hook also, just to be 100% that on delete it cleans it.
Let me just modify the script a bit

Reply:
Done